internal/documentdb/documentdb_api/documentdb_api.go

```diff
@@ -240,13 +240,13 @@ func DropCollection(ctx context.Context, conn *pgx.Conn, l *slog.Logger, databas
 
 // DropDatabase is a wrapper for
 //
-//	documentdb_api.drop_database(p_database_name text, p_write_concern documentdb_core.bson DEFAULT NULL, OUT drop_database void).
-func DropDatabase(ctx context.Context, conn *pgx.Conn, l *slog.Logger, databaseName string, writeConcern wirebson.RawDocument) (outDropDatabase string, err error) {
+//	documentdb_api.drop_database(p_database_name text, p_write_concern documentdb_core.bson DEFAULT NULL).
+func DropDatabase(ctx context.Context, conn *pgx.Conn, l *slog.Logger, databaseName string, writeConcern wirebson.RawDocument) (err error) {
 	ctx, span := otel.Tracer("").Start(ctx, "documentdb_api.drop_database", oteltrace.WithSpanKind(oteltrace.SpanKindClient))
 	defer span.End()
 
-	row := conn.QueryRow(ctx, "SELECT drop_database FROM documentdb_api.drop_database($1, $2::bytea)", databaseName, writeConcern)
-	if err = row.Scan(&outDropDatabase); err != nil {
+	row := conn.QueryRow(ctx, "SELECT FROM documentdb_api.drop_database($1, $2::bytea)", databaseName, writeConcern)
+	if err = row.Scan(); err != nil {
```

Comment:
Using `row.Scan()` without any argument for void function retrieves an error if any. Alternatively, we can use `conn.Exec(...)`.